include/constants/battle_config.h

```diff
@@ -144,6 +144,9 @@
 #define B_CRITICAL_CAPTURE          TRUE  // If set to TRUE, Critical Capture will be enabled.
 #define B_CATCHING_CHARM_BOOST      20    // % boost in Critical Capture odds if player has the Catching Charm.
 
+// Item Theft Settings
+#define B_TRAINERS_KNOCK_OFF_ITEMS  TRUE  // If TRUE, trainers can steal/swap your items (non-berries are restored after battle). In vanilla games trainers cannot steal items.
```

Comment:
This should maybe be turned into a Generational config because Gen 2-4 doesn't allow wild mons or regular in-game trainers to steal items apparently and Gen 5+ allows it. The config name should also be changed to something more like `B_TRAINERS_STEAL_ITEMS`.